components/localize/localize-mixin.js

```diff
@@ -0,0 +1,158 @@
+import IntlMessageFormat from 'intl-messageformat/src/main.js';
+window.IntlMessageFormat = IntlMessageFormat;
+
+export const LocalizeMixin = superclass => class extends superclass {
+
+	static get properties() {
+		return {
+			language: {type: String},
+			langResources: { type: Object },
+			__documentLanguage: { type: String },
+			__documentLanguageFallback: { type: String }
+		};
+	}
+
+	connectedCallback() {
+		if (super.connectedCallback) {
```

Comment:
Not that checking is a bad thing, but will this always be true?

Reply:
yeah I think this check is probably unnecessary